packages/input-edit/src/components/InputEditSearch/InputEditSearch.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import SimpleFormApp from 'tocco-simple-form/src/main'
+import _debounce from 'lodash/debounce'
+import {tqlBuilder} from 'tocco-util'
+import PropTypes from 'prop-types'
+
+const InputEditSearch = ({form, model, setSearchFields}) => {
+  if (form.children && model.paths) {
+    return <SimpleFormApp form={form}
+      model={model}
+      onChange={_debounce(handleChange(form, setSearchFields), 500)}
+      noButtons={true}/>
+  } else {
+    return null
+  }
+}
+
+const handleChange = (form, setSearchFields) => ({values, valid}) => {
+  if (valid && Object.keys(values).length > 0) {
+    const tql = Object.entries(values)
+      .map(([path, value]) => {
+        const parsedPath = path.replace('--', '.')
+        return {
+          path: parsedPath,
+          fieldType: loadFieldType(parsedPath, form),
+          value: value
+        }
+      })
+      .map(({path, fieldType, value}) => tqlBuilder.getTql(path, value, fieldType))
+    setSearchFields(tql)
+  }
+}
+
+const loadFieldType = (path, form) => {
```

Comment:
i usually name functions like this "getFieldType" since nothing is loaded. sorry 😄